extractors/pornhub/pornhub.go

```diff
@@ -0,0 +1,73 @@
+package pornhub
+
+import (
+	"encoding/json"
+	"github.com/iawia002/annie/downloader"
+	"github.com/iawia002/annie/request"
+	"github.com/iawia002/annie/utils"
+)
+
+type pornhubData struct {
```

Comment:
go fmt

Reply:
`go fmt` didn't change anything for me somehow.